src/main/java/core/basesyntax/Main/Main.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.Main;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+
+public class Main {
+    public static void main(String[] args) {
+        RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+        User user1 = new User(10l, "Mykola", "1234567", 21);
```

Comment:
The 'main' method is not needed for testing purposes as per the checklist: 'You don't need the main method to see how your solution works, you have tests for this purpose.' Consider removing this method.

Reply:
The 'main' method is not needed for testing purposes as per the checklist: 'You don't need the main method to see how your solution works, you have tests for this purpose.' Consider removing this method.